### PR TITLE
Addition of debug info file & examples folder

### LINKED ISSUE #135: Lamps not identified as lamps with f/w 1.3.14
I am a new user of pytradfri trying to integrate my gateway with homeassistant.
As I know HA only use version 4.1.0 of pytradfri I have also been testing with the standalone version of pytradfri 5.2.0

As my gateway is new and autoupgraded to v 1.3.14 imediately I have'nt tested any earlier firmwares of the gateway. My lamps have fw-version 1.2.217.

When I run pytradfri in Homeassistant my lamps are detected as sensors. I have the same problem with my standalone 5.2.0 test.

output from "python3 -i -m pytradfri 192.168.1.7 key"

```
DEBUG:pytradfri.api.libcoap_api:Executing 192.168.1.7 get ['15001']
DEBUG:pytradfri.api.libcoap_api:Received: [65536,65537,65539,65538,65540]
DEBUG:pytradfri.api.libcoap_api:Executing 192.168.1.7 get ['15001', 65536]
DEBUG:pytradfri.api.libcoap_api:Received: {"9001":"Fjärrkontroll 1","5750":0,"9002":1515435028,"9020":1515870426,"9003":65536,"9054":0,"9019":1,"3":{"0":"IKEA of Sweden","1":"TRADFRI remote control","2":"","3":"1.2.214","6":3,"9":87},"15009":[{"9003":0}]}
DEBUG:pytradfri.api.libcoap_api:Executing 192.168.1.7 get ['15001', 65537]
DEBUG:pytradfri.api.libcoap_api:Received: {"9001":"Tak vr 1","5750":0,"9002":1515435088,"9020":1516008067,"9003":65537,"9054":0,"9019":1,"3":{"0":"IKEA of Sweden","1":"TRADFRI bulb GU10 WS 400lm","2":"","3":"1.2.217","6":1},"15009":[{"5851":10,"5850":0,"5717":0,"5711":450,"5709":30138,"5710":26909,"5706":"efd275","9003":0}]}
DEBUG:pytradfri.api.libcoap_api:Executing 192.168.1.7 get ['15001', 65539]
DEBUG:pytradfri.api.libcoap_api:Received: {"9001":"Tak vr 3","5750":0,"9002":1515435277,"9020":1516008127,"9003":65539,"9054":0,"9019":1,"3":{"0":"IKEA of Sweden","1":"TRADFRI bulb GU10 WS 400lm","2":"","3":"1.2.217","6":1},"15009":[{"5851":10,"5850":0,"5717":0,"5711":450,"5709":30138,"5710":26909,"5706":"efd275","9003":0}]}
DEBUG:pytradfri.api.libcoap_api:Executing 192.168.1.7 get ['15001', 65538]
DEBUG:pytradfri.api.libcoap_api:Received: {"9001":"Tak vr 2","5750":0,"9002":1515435198,"9020":1516008063,"9003":65538,"9054":0,"9019":1,"3":{"0":"IKEA of Sweden","1":"TRADFRI bulb GU10 WS 400lm","2":"","3":"1.2.217","6":1},"15009":[{"5851":10,"5850":0,"5717":0,"5711":450,"5709":30138,"5710":26909,"5706":"efd275","9003":0}]}
DEBUG:pytradfri.api.libcoap_api:Executing 192.168.1.7 get ['15001', 65540]
DEBUG:pytradfri.api.libcoap_api:Received: {"9001":"Tak vr 4","5750":0,"9002":1515435391,"9020":1516008285,"9003":65540,"9054":0,"9019":1,"3":{"0":"IKEA of Sweden","1":"TRADFRI bulb GU10 WS 400lm","2":"","3":"1.2.217","6":1},"15009":[{"5851":10,"5850":0,"5717":0,"5711":450,"5709":30138,"5710":26909,"5706":"efd275","9003":0}]}
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.6/dist-packages/pytradfri/__main__.py", line 36, in <module>
    light = lights[0]
IndexError: list index out of range
>>> lights
[]
>>>
```

I would expect my 4 TRADFRI bulb GU10 WS 400lm to be identified as lamps, but the lights array looks empty. I've tried to read the source, but is new to python (c# and perl nerd).

    gateway = Gateway()
    devices_commands = api(gateway.get_devices())
    devices = api(devices_commands)
    lights = [dev for dev in devices if dev.has_light_control]
    light = lights[0]
 
It looks to me as pytradfri when calling the coap api does'nt get it right. The lamp devices does not have "has_light_control" set to true...

### PR DESCRIPTION
**Pull request by Lakitna**
As discussed at the end of #135 

The solution I've used works using the default import methods, but it's a hack. So don't use this solution in production code if it can be helped.

Closes #135

### CODE REVIEW

**Comment by ggravlingen** [comment]
Thank you very much! I'm fine with this 'hack' as it cleans up the root dir.

**Comment by ggravlingen** [comment]
@Lakitna perhaps also update the README as it references the examples?

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [![Coverage Status](https://coveralls.io/repos/github/ggravlingen/pytradfri/badge.svg?branch=master)](https://coveralls.io/github/ggravlingen/pytradfri?branch=master)
 [![PyPI version](https://badge.fury.io/py/pytradfri.svg)](https://badge.fury.io/py/pytradfri)
 
-**NB:** Latest Gateway version tested and working - 1.2.42.
```
Comment (Lakitna) [inline README.md]: Didn't properly rebase my fork before starting working on this. I rebased readme manually, this is an artifact of that.

**Comment by Lakitna** [comment]
@ggravlingen I agree, I've updated readme. I think that makes it ready to merge.

**Comment by ggravlingen** [comment]
Thank you very much @Lakitna!

**Pull request merged**